bundle/src/main/java/com/cognifide/cq/cqsm/core/loader/ScriptInclusion.java

```diff
@@ -0,0 +1,28 @@
+package com.cognifide.cq.cqsm.core.loader;
+
+import com.cognifide.apm.antlr.ApmLangParser.ScriptInclusionContext;
+import com.cognifide.cq.cqsm.core.antlr.StringLiteral;
+
+public final class ScriptInclusion {
+
+  private final String path;
+  private final ScriptInclusionContext inclusion;
+
+  public ScriptInclusion(String path, ScriptInclusionContext inclusion) {
+    this.path = path;

```

Comment:
You should concider using lombok annotations @Constructor and @Getter

Reply:
Fixed.

Reply:
Fixed.